libs/api/domains/vehicles/src/lib/services/vehicles.service.ts

```
@@ -301,13 +302,32 @@
+
+    const operatorAnonymityStatus = operatorStatusMapper(
+      operatorNames,
+      !!vehicle.allOperatorsAreAnonymous,
+      !!vehicle.someOperatorsAreAnonymous,
+    )
+
+    return {
+      ...data[0],
+
+      operatorNames:
+        operatorNames && operatorNames.length ? operatorNames : undefined,
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)